src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -100,6 +100,19 @@ import type { GraphMinimapDaySelectedEventDetail } from './minimap/minimap';
 import { GlGraphMinimapContainer } from './minimap/minimap-container.react';
 import { GlGraphSideBar } from './sidebar/sidebar.react';
 
+function getRemoteIcon(type: string | number) {
+	switch (type) {
+		case 'head':
+			return 'vm';
+		case 'remote':
+			return 'cloud';
+		case 'tag':
+			return 'tag';
+		default:
+			return '';
+	}
+}
```

Comment:
We should be using the remote's avatar icon for a remote branch in this list if the avatars are enabled in the graph's settings. Always using a generic cloud icon feels like a regression

![image](https://github.com/user-attachments/assets/7a351b68-f2e8-43ad-b318-49780f21f49d)